lib/pages/news_feed/view/news_feed_page.dart

```diff
@@ -24,6 +24,15 @@ class NewsFeedPage extends StatelessWidget {
           }
 
           final List<NewsFeedItem> newsFeedItems = snapshot.data;
+          if (newsFeedItems == null || newsFeedItems.isEmpty) {
+            return Center(
+                child: Text(
+                  S.of(context).warningNoneYet,

```

Comment:
This is kind of confusing for the user. If I see "none yet", I'll think there are no news to show, but actually - the problem is I don't have an internet connection. We should show a toast or something that actually warns the user that they don't have an internet connection.